app/controllers/api/v1/users_controller.rb

```diff
@@ -37,6 +37,7 @@ def index
       def create_and_invite_many
         users_attributes.each do |attrs|
           user_attributes = attrs.except(:invitation)
+          convert_tags_for_upsert_service!(user_attributes)
```

Comment:
pareil ici, on pourrait mettre ça directement dans la méthode `users_attributes`
```suggestion
        def users_attributes
        users_params.map do |user_attributes|
          user_attributes.to_h.deep_symbolize_keys.merge(creation_origin_attributes).tap do |attrs| 
              attrs[:invitation] ||= {}
              convert_tags_to_add_to_tag_users_attributes(attrs) 
           end
        end
      end

Reply:
vu ensemble